js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/EvaluationListRow.tsx

```diff
@@ -67,16 +70,28 @@ interface EvaluationRunInfoProps {
 
 const EvaluationRunInfo = ({runIds, timestamp}: EvaluationRunInfoProps) => {
   const [isOpen, setIsOpen] = useState(false);
+  const firstRun = runIds[0];
 
-  if (runIds.length === 0) {
+  if (!firstRun) {
     return <span style={{color: Colors.textDisabled()}}>None</span>;
   }
 
   if (runIds.length === 1) {
+    const truncated = firstRun.slice(0, 8);
+
+    // This looks like a backfill ID. Link there.
+    if (truncated === firstRun) {
+      return (
+        <Link to={`/runs/b/${firstRun}`}>
+          <Mono>{firstRun}</Mono>
+        </Link>
+      );
+    }
+
     return (
-      <Box flex={{direction: 'row', gap: 4}}>
-        <Mono>{runIds[0]}</Mono>
-      </Box>
+      <Link to={`/runs/${firstRun}`}>
+        <Mono>{truncated}</Mono>
+      </Link>
```

Comment:
It would be better to link to the Runs page filtering to all of the runs. 

Here is an example:
https://github.com/dagster-io/dagster/blob/5d205598e66fcf93b3973e7e2ea7d0acea7ef183/js_modules/dagster-ui/packages/ui-core/src/runs/RunUtils.tsx#L185-L189

Reply:
The linked example is kinda goofy though, I bet adding an array param to URLSearchParams would work.

Reply:
Hm, why is this better than linking directly to the run itself?

Reply:
If there is more than 1 run then there's no way to find the other runs?

Reply:
This is the branch where `runIds.length === 1`. If there are more runs, the dialog in the code below is available to show the filtered runs feed view.

Reply:
oh derp, missed that. carry on.